package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,777 @@
+            >=  `angle`. The default of 120º also finds fairly non-specific
+            hydrogen interactions and a possibly better value is 150º. [120.0]
+        forcefield : {"CHARMM27", "GLYCAM06", "other"} (optional)
+            Name of the forcefield used. Switches between different
+            :attr:`~HydrogenBondAnalysis.DEFAULT_DONORS` and
+            :attr:`~HydrogenBondAnalysis.DEFAULT_ACCEPTORS` values.
+            ["CHARMM27"]
+        donors : sequence (optional)
+            Extra H donor atom types (in addition to those in
+            :attr:`~HydrogenBondAnalysis.DEFAULT_DONORS`), must be a sequence.
+        acceptors : sequence (optional)
+            Extra H acceptor atom types (in addition to those in
+            :attr:`~HydrogenBondAnalysis.DEFAULT_ACCEPTORS`), must be a
+            sequence.
+        start : int (optional)
+            starting frame-index for analysis, ``None`` is the first one, 0.
+            `start` and `stop` are 0-based frame indices and are used to slice
+            the trajectory (if supported) [``None``]
+        stop : int (optional)
+            last trajectory frame for analysis, ``None`` is the last one
+            [``None``]
+        step : int (optional)
+            read every `step` between `start` (included) and `stop` (excluded),
+            ``None`` selects 1. [``None``]
+        distance_type : {"hydrogen", "heavy"} (optional)
+            Measure hydrogen bond lengths between donor and acceptor heavy
+            attoms ("heavy") or between donor hydrogen and acceptor heavy
+            atom ("hydrogen"). If using "heavy" then one should set the
+            *distance* cutoff to a higher value such as 3.5 Å. ["hydrogen"]
+        debug : bool (optional)
+            If set to ``True`` enables per-frame debug logging. This is disabled
+            by default because it generates a very large amount of output in
+            the log file. (Note that a logger must have been started to see
+            the output, e.g. using :func:`MDAnalysis.start_logging`.)
+        verbose : bool (optional)
+            Toggle progress output. (Can also be given as keyword argument to
+            :meth:`run`.)
+
+        Notes
+        -----
+        In order to speed up processing, atoms are filtered by a coarse
+        distance criterion before a detailed hydrogen bonding analysis is
+        performed (`filter_first` = ``True``).
+
+        If selection 1 and selection 2 are very mobile during the simulation and the contact surface is constantly changing (i.e. residues are  moving farther than 3 x `distance`), you might consider setting the
+        `update_selection1` and `update_selection2` keywords to ``True`` to ensure correctness.
+        """
+        self.water_selection = water_selection
+        self.update_water_selection = update_water_selection
+        HydrogenBondAnalysis.__init__(self, universe=universe,
```

Comment:
Not good Python oo style. Use `super()`:
```
super(WaterBridgeAnalysis, self).__init__(universe=..., ...)
```

Reply:
@orbeckst Thank you. I have changed it to super().__init__()